### PR TITLE
[new] [author] Add webpack-cesium@1.31.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by TJKoury**
Pull request for issue: #11442
Related issue(s): None

# Profile of the lib
 * Git repository (required): 
    - https://github.com/AnalyticalGraphicsInc/cesium
    - https://github.com/DigitalArsenal/webpack-cesium
 * Official website (optional, not the repository): https://cesiumjs.org
 * NPM package url: 
    - https://www.npmjs.com/package/webpack-cesium
    - https://www.npmjs.com/package/cesium
 * License and its reference: ISC
    - [Apache 2](http://www.apache.org/licenses/LICENSE-2.0.html)
 * GitHub / Bitbucket popularity: 
   - Count of watchers: 268
   - Count of stars: 2,268
   - Count of forks: 1,092
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by dakshshah96** [comment]
Hey @TJKoury,

This library doesn't seem to meet the minimum popularity requirements.

cc @PeterDaveHello

**Comment by TJKoury** [comment]
@dakshshah96 Check out the discussion in the issue referenced in the pull.

**Comment by dakshshah96** [comment]
@TJKoury Awesome! No issues then 😄

File: ajax/libs/webpack-cesium/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "wms",
+    "wmts",
+    "kml",
+    "czml",
+    "space",
+    "satellite",
+    "agi",
+    "stk"
+  ],
+  "author": "digitalarsenal.io",
```
Comment (dakshshah96) [inline ajax/libs/webpack-cesium/package.json]: If this library was authored by a team, you can change this to `DigitalArsenal Team (http://digitalarsenal.io)`.
If it was authored a single person, you can mention his/her name here.
If you'd like to specify multiple authors you can use `"authors"` here.

File: ajax/libs/webpack-cesium/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webpack-cesium",
+  "version": "1.31.2",
+  "description": "webpack version of Cesium",
+  "filename": "webpack.cesium.min.js",
```
Comment (dakshshah96) [inline ajax/libs/webpack-cesium/package.json]: You can use `webpack.cesium.js` here since it is already minified.

**Comment by TJKoury** [comment]
I think I got it all @dakshshah96, please advise and thanks for bearing with me.

**Comment by TJKoury** [comment]
Fixing!  Looks like I squashed an additional commit by mistake...

**Comment by PeterDaveHello** [comment]
@TJKoury let me know if you need help 😄 
cc @cdnjs/intern3 @cdnjs/active-contributor

**Comment by PeterDaveHello** [comment]
Looks like we moved to #11461

**Pull request closed without merging**